MHVLib/Types/MHVItem.m

```diff
@@ -90,7 +90,7 @@ - (void)setNote:(NSString *)note
 
 - (MHVBlobPayload *)blobs
 {
-    MHVENSURE(_blobs, [MHVBlobPayload class]);
+    MHVENSURE(_blobs, MHVBlobPayload);
```

Comment:
We should get rid of this macro - It hides some pretty important code. We don't need to remove it everywhere in the project right now, but we should start deleting it when we touch a file.